sqsworkers/crew.py

```diff
@@ -177,7 +185,8 @@ def poll_queue(self):
                                 }]
                             )
                             self.crew.statsd.increment('process.record.success', 1, tags=[])
-                            self.logger.info('%s messages processed successfully and deleted %s' % (len(messages), deleted))
+                            self.logger.info('%s messages processed successfully and deleted %s' %
```

Comment:
It's probably better to replace these with `str.format`, but it's just a suggestion

Reply:
There are several places where this is used. If we replace, it should be for all. But I would like to focus on our objective for this PR here and avoid rabbit holes that I can avoid